src/add-group-modal/AddGroupModal.tsx

```diff
@@ -48,30 +47,111 @@ const NewGroupForm = () => {
         rowGap: "1rem",
       }}
     >
-      <NameField />
+      <TextInput
+        labelText={t("newGroupName", "New Group Name")}
+        value={name}
+        onChange={(e) => setName(e.target.value)}
+        onBlur={() => validate("name")}
+      />
+      {errors?.name && (
+        <p className={styles.formError}>Please enter a group name.</p>
+      )}
       <FormLabel>Patients in group</FormLabel>
-      <ul>
-        {patientList?.map((patient, index) => (
-          <li key={index}>{patient?.display}</li>
-        ))}
-      </ul>
+      {errors?.patientList && (
+        <p className={styles.formError}>Please enter a at least one patient.</p>
```

Comment:
```suggestion
        <p className={styles.formError}>Please enter at least one patient.</p>
```

If there's time for this, we could also use `t` in this file, but I'd not consider it super important for the moment.